vignettes/bisg.Rmd

```diff
@@ -3,53 +3,80 @@ title: "Bayesian Improved Surname Geocoding (BISG)"
 author: "null"
 date: "8/5/2020"
 output:
-  pdf_document: default
   html_document: default
+  pdf_document: default
 ---
 This vignette demonstrates how to perform Bayesian Improved Surname Geocoding when the race/ethncity of individuals are unknown within a dataset.
 
 ## *What is Bayesian Improved Surname Geocoding?*
 
 Bayesian Improved Surname Geocoding (BISG) is a method that applies the Bayes Rule/Theorem to predict the race/ethnicity of an individual using the individual's surname and geocoded location [Elliott et. al 2008, Elliot et al. 2009, Imai and Khanna 2016]. 
 
-Specifically, BISG first calculates the prior probability of *i* individual being of a ceratin *r* racial group given their *s* surname, or p(r~i~|s~i~). The prior probability created from the surname is updated using Bayes Theorem using the probability of the *i* individual *r* race given their *g* geographic location. The following equation describes how BISG calculates race/ethnicity of individuals when race/ethncicty is unknown using Bayes Theorem:
+Specifically, BISG first calculates the prior probability of individual *i* being of a ceratin racial group *r* given their surname *s* or $$Pr(R_i=r|S_i=s)$$. The prior probability created from the surname is then updated with the probability of the individual *i* living in a geographic location *g* belonging to a racial group *r*, or $$Pr(G_i=g|R_i=r)$$). The following equation describes how BISG calculates race/ethnicity of individuals using Bayes Theorem, given the surname and geographic location, and specifically when race/ethncicty is unknown :
+
 
-      ![BISG Equation for Predicting Race/Ethnicity.](/github/eiCompare/vignettes/bisg_equation.png)
+$$Pr(R_i=r|S_i=s, G_i=g)=\frac{Pr(G_i= g|R_i =r)Pr(R_i =r |S_i= s)}{\sum_{i=1}^n Pr(G_i= g|R_i =r)Pr(R_i =r |S_i= s)}$$
 
-In R, the package that performs BISG is called, WRU: Who Are You `wru` [cite WRU package]. This vignette will walk you through how to prepare your geocoded voter file for performing BISG by stepping you throught the processing of cleaning your voter file, prepping voter data for running the BISG, and finally, performing BISG to obtain racial/ethnic probailities of individuals in a voter file.
+  
+In R, the package that performs BISG is called, WRU: Who Are You `wru` [https://cran.r-project.org/web/packages/wru/index.html]. This vignette will walk you through how to prepare your geocoded voter file for performing BISG by stepping you throught the processing of cleaning your voter file, prepping voter data for running the BISG, and finally, performing BISG to obtain racial/ethnic probailities of individuals in a voter file.
```

Comment:
typo on "stepping you through"

Reply:
"the **process** of cleaning your voter file"

Reply:
thanks!